cranelift/codegen/src/opts/algebraic.isle

```diff
@@ -519,21 +467,3 @@
 (rule (simplify
        (select ty (fcmp _ (FloatCC.GreaterThan) x y) x y))
       (fmax_pseudo ty x y))
-
-;; Do the same for vectors
-(rule (simplify
-       (vselect ty (fcmp _ (FloatCC.LessThan) x y) x y))
-      (fmin_pseudo ty x y))
-(rule (simplify
-       (vselect ty (fcmp _ (FloatCC.GreaterThan) x y) x y))
-      (fmax_pseudo ty x y))
-
-;; If both of the multiplied arguments to an `fma` are negated then remove
-;; both of them since they cancel out.
-(rule (simplify (fma ty (fneg ty x) (fneg ty y) z))
```

Comment:
I think these were removed by mistake right?

Reply:
Oh dear now I'm not sure how these leaked in...